src/commands/utility/assignTaskForum.js

```diff
@@ -65,7 +66,7 @@ module.exports = {
 
       const forum = channel.parent;
       const assignedUser = (
-        options.getString('user') || user.username
+        options.getUser('user').username || user.username
```

Comment:
It will not break if user is not selected?, not sure if we need add a optional chaining here, what do you think?
```suggestion
        options.getUser('user')?.username || user.username
```